resources/public/api/conf/6.0/bsas_trigger.yaml

```
@@ -35,18 +35,30 @@ post:
         schema:
           $ref: "./schemas/trigger/request.json"
         examples:
-          example-1:
+          PRE-TY-2024-25-Self-Employment:
             description: Self-employment Trigger
```

Would it be worth spelling out that the tax year in the description? I.e...
```suggestion
            description: Self-employment Trigger for TY 2023-24 and before
```